src/rev_ai/models/asynchronous/transcript.py

```diff
@@ -27,35 +27,71 @@ def from_json(cls, json):
 
 
 class Monologue:
-    def __init__(self, speaker, elements):
+    def __init__(self, speaker, speaker_info, elements):
```

Comment:
I don't think you should inject new parameters in the middle of the initializer. Theoretically someone could create a Monologue object and this would break their implementation.


can you pass None as default value for the initializer?